plugins/azure/fix_plugin_azure/resource/storage.py

```diff
@@ -0,0 +1,1078 @@
+                name=sku_name,
+                tier=sku_tier,
+                clazz=AzureStorageSku,
+            )
+
+
+@define(eq=False, slots=False)
+class AzureStorageUsage(AzureResource, AzureBaseUsage):
+    kind: ClassVar[str] = "azure_storage_usage"
+    api_spec: ClassVar[AzureApiSpec] = AzureApiSpec(
+        service="storage",
+        version="2023-01-01",
+        path="/subscriptions/{subscriptionId}/providers/Microsoft.Storage/locations/{location}/usages",
+        path_parameters=["subscriptionId", "location"],
+        query_parameters=["api-version"],
+        access_path="value",
+        expect_array=True,
+        expected_error_codes=AzureBaseUsage._expected_error_codes,
+    )
+    mapping: ClassVar[Dict[str, Bender]] = AzureBaseUsage.mapping | {
+        "id": S("name", "value"),
+    }
+    _is_provider_link: bool = False
+
+
+@define(eq=False, slots=False)
+class AzureTableAccessPolicy:
+    kind: ClassVar[str] = "azure_table_access_policy"
+    mapping: ClassVar[Dict[str, Bender]] = {
+        "expiry_time": S("expiryTime"),
+        "permission": S("permission"),
+        "start_time": S("startTime"),
+    }
+    expiry_time: Optional[datetime] = field(default=None, metadata={"description": "Expiry time of the access policy"})
+    permission: Optional[str] = field(default=None, metadata={'description': 'Required. List of abbreviated permissions. Supported permission values include r , a , u , d '})  # fmt: skip
+    start_time: Optional[datetime] = field(default=None, metadata={"description": "Start time of the access policy"})
+
+
+@define(eq=False, slots=False)
+class AzureTableSignedIdentifier:
+    kind: ClassVar[str] = "azure_table_signed_identifier"
+    mapping: ClassVar[Dict[str, Bender]] = {
+        "access_policy": S("accessPolicy") >> Bend(AzureTableAccessPolicy.mapping),
+        "id": S("id"),
+    }
+    access_policy: Optional[AzureTableAccessPolicy] = field(default=None, metadata={'description': 'Table Access Policy Properties Object.'})  # fmt: skip
+    id: Optional[str] = field(default=None, metadata={'description': 'unique-64-character-value of the stored access policy.'})  # fmt: skip
+
+
+@define(eq=False, slots=False)
+class AzureTable(AzureResource):
+    kind: ClassVar[str] = "azure_table"
+    mapping: ClassVar[Dict[str, Bender]] = {
+        "id": S("id"),
+        "tags": S("tags", default={}),
+        "name": S("name"),
+        "table_signed_identifiers": S("properties", "signedIdentifiers")
+        >> ForallBend(AzureTableSignedIdentifier.mapping),
+        "table_name": S("properties", "tableName"),
+    }
+    table_signed_identifiers: Optional[List[AzureTableSignedIdentifier]] = field(default=None, metadata={'description': 'List of stored access policies specified on the table.'})  # fmt: skip
+    table_name: Optional[str] = field(default=None, metadata={"description": "Table name under the specified account"})
+
+
+resources: List[Type[AzureResource]] = [
```

Comment:
Please list all resource types here: AzureBlobContainer, etc. is missing
(even if it is collected via storage account)